data/nginx-relay/nginx.conf

```diff
@@ -21,6 +21,9 @@ stream {
         svr2.signal.org                         svr2;
         updates.signal.org                      updates;
         updates2.signal.org                     updates2;
+        backend1.svr3.signal.org                svr31;
+        backend2.svr3.signal.org                svr32;
+        backend3.svr3.signal.org                svr33;
```

Comment:
Unfortunately this list isn’t strictly alphabetized or grouped, but these could move up with svr2.